metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/rest/client/enrichment/EnricherImpl.java

```diff
@@ -4,23 +4,25 @@
 
 import eu.europeana.corelib.definitions.jibx.RDF;
 import eu.europeana.enrichment.api.external.model.EnrichmentBase;
-import eu.europeana.enrichment.api.external.model.EnrichmentBaseWrapper;
+import eu.europeana.enrichment.api.external.model.EnrichmentResultBaseWrapper;
 import eu.europeana.enrichment.api.external.model.EnrichmentResultList;
 import eu.europeana.enrichment.rest.client.exceptions.EnrichmentException;
 import eu.europeana.enrichment.utils.EnrichmentFields;
 import eu.europeana.enrichment.utils.EnrichmentUtils;
 import eu.europeana.enrichment.utils.EntityMergeEngine;
-import eu.europeana.enrichment.utils.InputValue;
+import eu.europeana.enrichment.api.external.SearchValue;
+import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 import java.util.stream.Collectors;
 import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.math3.util.Pair;
```

Comment:
I think it is more common to use `org.apache.commons.lang3.tuple.Pair` instead, but I don't think it matters. What does matter is consistency: whatever you use elsewhere you should use here.

Reply:
I agree with you, consistency matters and we should use the same everywhere.
I didn't know that `org.apache.commons.lang3.tuple.Pair` is more common, I will change it